Assets/MixedRealityToolkit.Extensions/SceneTransitionService/Scripts/LoadContentScene.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.SceneSystem;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
```

Comment:
Is it expected that this will be used by non-example code?

Reply:
Yes, it is a common utility script for using Scene System.
@Railboy This script should work even if the Scene Transition Service is not registered?

Reply:
@cre8ivepark That was originally this case, but this script now works exclusively with the transition service extension.

@chrisfromwork This script can be used by non-example code.

This touches on a larger discussion about how extension services are consumed / how to resolve assembly references between the SDK and extensions. Right now there isn't a clean way for SDK scripts to reference extensions.

Reply:
Now it makes sense having this script under SceneTransitionService which uses it.